tests/test_tools/testing.py

```diff
@@ -0,0 +1,145 @@
+import io
+import contextlib
+import pathlib
+import re
+import sys
+
+import typing as t
+
+from click import termui, testing, types
+
+
+from incipyt import project
+
+
+def diff_files(dircmp):
+    result = {str(pathlib.Path(dircmp.left) / f) for f in dircmp.diff_files}
+    for sd in dircmp.subdirs.values():
+        result |= diff_files(sd)
+    return {
+        path
+        for path in result
+        if not any(
+            re.match(pattern, path) for pattern in [r".*/\.env/.*", r".*/dist/.*"]
```

Comment:
let's use or own .gitignore https://stackoverflow.com/a/22090594

Reply:
while I agree those need to be stored somewhere else (a list of globs, just as .gitignore does, is just fine), using incipyt own .gitignore seems ankward and error-prone to me

Reply:
also, I'm not sure a 3rd party lib is really needed when we already have [glob](https://docs.python.org/3/library/glob.html) in the stdlib